### PR TITLE
[15.0] [MIG] storage_backend_s3

### PR DESCRIPTION
**Pull request by JasminSForgeFlow**
Standard Migration

@ForgeFlow

### CODE REVIEW

File: storage_backend_s3/README.rst [hunk 1]
```diff
@@ -0,0 +1,45 @@
+
+.. image:: https://img.shields.io/badge/licence-LGPL--3-blue.svg
+   :target: http://www.gnu.org/licenses/lgpl-3.0-standalone.html
+   :alt: License: LGPL-3
+
+===================
+Storage backend S3
+===================
+
+Add the possibility to store and get data from amazon S3 for your storage backend
```
Comment (MiquelRForgeFlow) [inline storage_backend_s3/README.rst:10]: Could you create the folder 'readme' and add some fragments in it? This way, the readme will be updated to OCA guidelines.

Comment (JasminSForgeFlow) [reply storage_backend_s3/README.rst:10]: Added,

Thanks

**Review by MiquelRForgeFlow — APPROVED** [review]

**Review by DavidJForgeFlow — APPROVED** [review]
LGTM!

File: storage_backend_s3/README.rst [hunk 2]
```diff
@@ -22,8 +22,6 @@ To install this module, you need to:
 Known issues / Roadmap
 ======================
 
-Update README with the last model of README when migration to v11 in OCA
-
 There is an issue with the latest version of `boto3` and `urllib3`
 - boto3 needs to be `boto3<=1.15.17` related with https://github.com/OCA/storage/issues/67
```
Comment (etobella) [inline storage_backend_s3/README.rst:26]: Can you add this to the manifest then? on 15 you can force this :smile:

Comment (MiquelRForgeFlow) [reply storage_backend_s3/README.rst:26]: Done!

**Review by etobella — APPROVED** [review]

**Pull request merged**